README.md

```diff
@@ -4,6 +4,128 @@ This repository contains a tiny implementation of the Bookstore gRPC service
 that can be deployed to Cloud Run or another platform to provide a simple
 target server that can be used to test gRPC API management systems and clients.
 
+## Developing the Bookstore API with the API Registry
+
+Here's a brief exercise that shows how the registry can be used to track and
```

Comment:
Should we consistently use the proper name "API Registry" instead of just "registry"?

Reply:
I'll fix this instance. Maybe we need a style guide.